docker-compose.yml

```diff
@@ -49,34 +49,6 @@ services:
       # https://developers.google.com/recaptcha/docs/faq#id-like-to-run-automated-tests-with-recaptcha.-what-should-i-do
       - GOOGLE_CAPTCHA=6LeIxAcTAAAAAGG-vFI1TnRWxMZNFuojJ4WifJWe
       - GOOGLE_CAPTCHA_PUBLIC=6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI
-      - TWILIO_ACCOUNT_SID
```

Comment:
actually do we really need to delete these? even if we leave it in, env vars declared in `.env` will be injected correctly right? I'm asking because ultimately all of these will have local env defaults once we finish removing feature manager

Reply:
They won't actually; dotenv will not overwrite anything in `process.env` that is already set. I assume that stating the env var in docker-compose will implicitly set it.

Reply:
seems to me this isn't a big loss, apart from serving as documentation?

Reply:
yup not a big loss, actually mainly because I'm adding nearly all these env vars back anyway - see #2218 and #2222